ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -226,7 +220,7 @@ Column {
     Component {
         id: invitationBubble
         InvitationBubbleView {
-            store: root.store
+            // store: root.store
```

Comment:
Can we get rid of this?

Reply:
I didn t really refactor that one and it needs a store. I don't know :)